### PR TITLE
Add some validations for L4 multinetwork services

### PR DESCRIPTION
**Pull request by mmamczur**
- only services in network type L3 are supported.
- multinet services with externalTrafficPolicy other than Local are not supported for now.

### CODE REVIEW

File: pkg/network/network.go [hunk 1]
```diff
@@ -73,6 +71,12 @@ func (nr *NetworksResolver) ServiceNetwork(service *apiv1.Service) (*NetworkInfo
 	if !ok || networkName == "" || networkName == networkv1.DefaultPodNetworkName {
 		return DefaultNetwork(nr.cloudProvider), nil
 	}
+
+	// TODO: remove this check once DPv2 supports externalTrafficPolicy=Cluster services.
```
Comment (alexkats) [inline pkg/network/network.go:75]: Shouldn't we use cilium here instead of DPv2?

Comment (alexkats) [reply pkg/network/network.go:75]: It'd good to point to some issue in cilium, if there's any

Comment (mmamczur) [reply pkg/network/network.go:75]: this is a google specific feature of multi networking, not available in vanilla cilium.

Comment (alexkats) [reply pkg/network/network.go:75]: right, sorry for the nit picking

**Pull request merged**